tests/index.test.js

```diff
@@ -162,7 +164,7 @@ tap.test('Plain Custom Service', test => {
   test.test('Can return a stream', async assert => {
     const filename = './tests/services/plain-custom-service.js'
     const readFile = promisify(fs.readFile)
-    const fastify = await setupFastify(baseEnv)
+    const fastify = await setupFastify({ ...baseEnv, MY_REQUIRED_ENV_VAR })
```

Comment:
Before this change only baseEnv was enough to setup fastify, why now also the new const has to be provided? This feels like we missed a really nasty bug or the PR introduces a breaking change 🤔

Reply:
This is due to the fact that I changed some generated schemas and created a new one.
It's not required to add this variable, it's just for the unitary test

Reply:
I removed these unused variables I introduced in some tests in which they were not useful (I refer to the `{...baseEnv, MY_REQUIRED_ENV_VAR}` part)